### PR TITLE
feat: SUDT acp account can destory when sUDT amount is 0

### PR DESCRIPTION
**Pull request by yanguoyu**
1. Destroy sUDT acp account

https://user-images.githubusercontent.com/12881040/164125278-3b8f7276-fd9e-46f4-81bf-bebacf7ea475.mov

2. Destroy CKB acp account

https://user-images.githubusercontent.com/12881040/164125331-0d2fadab-4e73-455f-b256-e48eaa765dcc.mov

### CODE REVIEW

File: packages/neuron-ui/src/services/remote/remoteApiWrapper.ts [hunk 1]
```diff
@@ -103,7 +103,7 @@ type Action =
   | 'migrate-acp'
   | 'check-migrate-acp'
   | 'get-sudt-token-info'
-  | 'generate-destroy-ckb-account-tx'
+  | 'generate-destroy-account-tx'
```
Comment (Keith-CY) [inline packages/neuron-ui/src/services/remote/remoteApiWrapper.ts:106]: `generate-destroy-asset-account-tx`

File: packages/neuron-wallet/src/services/tx/transaction-generator.ts [hunk 2]
```diff
@@ -704,29 +704,20 @@ export class TransactionGenerator {
...
-
-    const cellDeps = [secpCellDep, anyoneCanPayDep]
 
-    const { inputs: changeInputs } = await CellsService.gatherInputs('0', walletId)
-
-    if (changeInputs.length === 0) {
-      throw new CapacityNotEnough()
+    const cellDeps = [secpCellDep, assetAccountInfo.anyoneCanPayCellDep]
+    if (!isCKBAccount) {
+      cellDeps.push(assetAccountInfo.sudtCellDep)
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/services/tx/transaction-generator.ts:718]: Should verify if the balance of sudt is 0 in service.

File: packages/neuron-wallet/src/services/tx/transaction-generator.ts [hunk 3]
```diff
@@ -736,17 +727,34 @@ export class TransactionGenerator {
...
-    tx.fee = TransactionFee.fee(txSize, BigInt(1e4)).toString()
+    let allCapacities = asssetAccountInputs.reduce((a, b) => {
+      return a + BigInt(b.capacity as string)
+    }, BigInt(0))
+    tx.fee = TransactionFee.fee(TransactionSize.tx(tx), BigInt(1e4)).toString()
     const outputCapacity = allCapacities - BigInt(tx.fee)
     tx.outputs[0].capacity = outputCapacity.toString()
 
+    if (outputCapacity >= BigInt(MIN_CELL_CAPACITY)) {
+      tx.outputs[0].capacity = outputCapacity.toString()
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/services/tx/transaction-generator.ts:744]: It has been assigned in line 741

**Review by Keith-CY — APPROVED** [review]

**Comment by Keith-CY** [comment]
CI failed

**Comment by yanguoyu** [comment]
> CI failed

Fixed by https://github.com/nervosnetwork/neuron/pull/2337/commits/db76074b8ee4bcf307c284050be13ae9bb5d9b4c

**Review by Keith-CY — APPROVED** [review]

**Pull request merged**